layouts/_default/baseof.html

```diff
@@ -5,6 +5,7 @@
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
     <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
+    <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
```

Comment:
@elswork Could you group all commits to single PR. I saw you open another PR: https://github.com/huyb1991/hugo-lamp/pull/64
This should be 1 PR, and many commits.